scripts/build/gen_relocate_app.py

```diff
@@ -514,12 +520,13 @@ def parse_input_string(line):
         phdr, rest = rest.split(':', 1)
 
     # Split lists by semicolons, in part to support generator expressions
-    flag_list, file_list = (lst.split(';') for lst in rest.split(':', 1))
+    flag_list, file_list, symbol_filter = (lst.split(';') for lst in rest.split(':', 2))
```

Comment:
This fails on windows, most likely because drive letters contains `:`.

```
c:/users/torsten/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd.exe: app/libapp.a(test_file1.c.obj): in function `code_relocation_test_function_in_sram2':
C:/projects/github/ncs/zephyr/tests/application_development/code_relocation/src/test_file1.c:80: undefined reference to `__sram2_data_reloc_start'
c:/users/torsten/zephyr-sdk-0.16.8/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd.exe: C:/projects/github/ncs/zephyr/tests/application_development/code_relocation/src/test_file1.c:80: undefined reference to `__sram2_data_reloc_end'
....
```

Also note the existing warning in this script:
https://github.com/zephyrproject-rtos/zephyr/blob/878640fa0817a5c6684758d68dcb824e24571612/scripts/build/gen_relocate_app.py#L508

Reply:
@tejlmand do you have a way to test on windows without windows?

Reply:
unfortunately not.